### PR TITLE
Make it possible to run as a daemon.

### PR DESCRIPTION
**Pull request by rantan**

### CODE REVIEW

File: src/command_args.rs [hunk 1]
```diff
@@ -12,35 +12,47 @@ use serde::Deserialize;
...
 
+/// # Default Values
 pub const DEFAULT_RPC_HOST: &str = "127.0.0.1";
 pub const DEFAULT_RPC_PORT: &str = "2377";
 pub const DEFAULT_RPC_USERNAME: &str = "";
 pub const DEFAULT_RPC_PASSWORD: &str = "";
 pub const DEFAULT_REDIS_HOST: &str = "127.0.0.1";
 pub const DEFAULT_REDIS_PORT: &str = "6379";
 pub const DEFAULT_LOG_LEVEL: &str = "info";
+pub const DEFAULT_PID: &str = "/tmp/tapyrus-signer.pid";
```
Comment (azuchi) [inline src/command_args.rs]: The tmp directory path is OS dependent. It is better to get it with `std::env::temp_dir`.  `/var/log` too.

Comment (rantan) [reply src/command_args.rs]: Ok, I will fix it.

Comment (rantan) [reply src/command_args.rs]: Regarding log file, which place is better? I have no idea. so i'm thinking that i'm going to remove default value.

Comment (azuchi) [reply src/command_args.rs]: How about same `tmp` dir, instead of remove?

Comment (rantan) [reply src/command_args.rs]: It's ok too. I will.

**Pull request merged**